spec/terminal-unicode-core.tex

```diff
@@ -105,8 +105,8 @@ \section{Feature and Mode State Detection}
 
 \UCTEST \cite{DECRQM} can be used for testing the availability of this
 feature as well as the current mode the terminal is in with regards
-to this specification, the \UCTEST reply will indigate each state
-acurately enough not not need any new VT sequence introduced.
+to this specification, the \UCTEST reply will indicate each state
+accurately enough to not need any new VT sequence introduced.
```

Comment:
Possibly "not to" rather than "to no", but either is better than "not not" 😉

> ... will indicate each state accurately enough not to need any new VT sequence introduced.